examples/with-next-auth/.env.development.example

```diff
@@ -0,0 +1 @@
+SITE=http://localhost:3000
```

Comment:
This file should be `.env.development`.

Reply:
It is, isn't it?

Reply:
Without `.example`, that is because it's not `.env.development.local`, so it will be tracked by git 👍 

`.env*.local` = Not tracked by git as it may contain secrets
`.env` = Contains defaults that anyone with access to the repo may need

Reply:
Ahh okay gotcha :+1: